### ISSUE TITLE
Autocomplete variables

### ISSUE THREAD

**Issue by josephtocci** [workflowproducts/postage#199]
When I type `tablename.` and look at the autocomplete results, I get a bunch of variables from the function. In this particular case with the autocomplete, I only want column names.

![screen shot 2017-04-08 at 9 54 52 am](https://cloud.githubusercontent.com/assets/14278246/24829940/84523e72-1c41-11e7-855e-bc5100816376.png)

**Comment by justintocci** [comment]
Right, when we're dropping down on a schema, it shouldn't be including variable names.

**Comment by crosstocci** [comment]
The variable feature needs to be added to the new autocomplete.
